.github/workflows/.env/pr-tests/min_versions.env

```diff
@@ -0,0 +1 @@
+k8sgateway_api_version='v1.1.0'
```

Comment:
I initially set this value to `v1.0.0` to match `.github/workflows/.env/nightly-tests/min_versions.env` but the following conformance test consistently failed:

```sh
=== RUN   TestConformance/GatewayWithAttachedRoutes/Gateway_listener_should_have_AttachedRoutes_set_even_when_Gateway_has_unresolved_refs
    helpers.go:650: Gateway expected observedGeneration to be updated to 1 for all conditions, only 0/2 were updated. stale conditions are: Accepted (generation 0), Programmed (generation 0)
    helpers.go:655: Conditions matched expectations
    helpers.go:655: Gateway status listeners matched expectations
    helpers.go:681: Accepted condition set to Status True with Reason Accepted, expected Status False
    helpers.go:681: Accepted was not in conditions list [[{ResolvedRefs False 1 2024-12-11 00:21:38 +0000 UTC BackendNotFound Service "does-not-exist" not found} {Accepted True 1 2024-12-11 00:21:38 +0000 UTC Accepted }]]
```

^ is a test bug that was fixed in https://github.com/kubernetes-sigs/gateway-api/commit/8bd549108154a714b16d9e30b168b13200bb7b0d (included in v1.1.0). @sam-heilbron should `.github/workflows/.env/nightly-tests/min_versions.env` be updated accordingly?